### ISSUE TITLE
Unit tests fail on missing numpy.int

### ISSUE THREAD

**Issue by mplpl** [IBM/nzpyida#19]
When running unit tests from nzpyida/tests the following error occurs (massively!):
```
E           AttributeError: module 'numpy' has no attribute 'int'.
E           `np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
E           The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
E               https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

/Users/mpl/Library/Python/3.9/lib/python/site-packages/numpy/__init__.py:305: AttributeError

```

My version of numpy is:
```
>>> import numpy
>>> numpy.__version__
'1.24.3'
```

**Comment by mplpl** [comment]
numpy.int was deprecated in NumPy 1.20 and removed in 1.24.

**Comment by mplpl** [comment]
The problem occurs on calling _create_table() function from IdaDataBase.

```
idadb._create_table(pred_df[:10].as_dataframe(), 't1')
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[3], line 1
----> 1 idadb._create_table(pred_df[:10].as_dataframe(), 't1')

File [~/Documents/work/nzpyida/nzpyida/base.py:2084](https://file+.vscode-resource.vscode-cdn.net/Users/mpl/Documents/work/~/Documents/work/nzpyida/nzpyida/base.py:2084), in IdaDataBase._create_table(self, dataframe, tablename, primary_key)
   2082     column_string += "\"%s\" VARCHAR(255)," % str(column).strip()
   2083 else:
-> 2084     if dataframe.dtypes[column] in [np.int64, np.int, np.int8, np.int32]:
   2085         if abs(dataframe[column].max()) < 2147483647[/](https://file+.vscode-resource.vscode-cdn.net/)2: # might get bigger 
   2086             datatype = "INT"

File [~/Library/Python/3.9/lib/python/site-packages/numpy/__init__.py:305](https://file+.vscode-resource.vscode-cdn.net/Users/mpl/Documents/work/~/Library/Python/3.9/lib/python/site-packages/numpy/__init__.py:305), in __getattr__(attr)
    300     warnings.warn(
    301         f"In the future `np.{attr}` will be defined as the "
    302         "corresponding NumPy scalar.", FutureWarning, stacklevel=2)
    304 if attr in __former_attrs__:
--> 305     raise AttributeError(__former_attrs__[attr])
    307 # Importing Tester requires importing all of UnitTest which is not a
    308 # cheap import Since it is mainly used in test suits, we lazy import it
    309 # here to save on the order of 10 ms of import time for most users
    310 #
    311 # The previous way Tester was imported also had a side effect of adding
    312 # the full `numpy.testing` namespace
    313 if attr == 'testing':

AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

**Comment by mplpl** [comment]
Fixed in 0.3.3
